tests/test_backend_options.py

```diff
@@ -90,27 +90,27 @@ def circuit():
 
                 log.info("Outcome: %s", measurement)
 
-    def test_basicaer_chop_threshold(self):
-        """Test BasisState with preparations on the whole system."""
-        if self.devices is None:
-            return
-        self.logTestName()
+    # def test_basicaer_chop_threshold(self):
```

Comment:
Is this test supposed to be commented out?

Reply:
A bit embarrassing, but I spent a good couple of hours attempting to debug this test, but (a) couldn't work out what it was supposed to do, and (b) couldn't work out why any of my changes broke it. I figured with your test refactor PR happening at the same time, we could replace it and then see what happened :laughing: